os/Kconfig.debug

```diff
@@ -167,6 +167,17 @@ config DEBUG_MM_INFO
 	---help---
 		Enable memory management informational debug SYSLOG output.
 
+config DEBUG_DOUBLE_FREE
+	bool "Debug Double Free Attempt"
+	default n
+	depends on DEBUG
```

Comment:
These are already under DEBUG, line 17 so that this is duplicated. Let's remove "depends on DEBUG".

Reply:
done